### ISSUE TITLE
Enhance the item-, monster- and conversation editors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#154]
```
Currently, the editors for items, monsters and conversation are quick ugly 
hacks that I made a while ago to get things going. These editors manage to do 
the job, but are really cumbersome to work with sometimes. Currently, they are 
implenented in pure html and jQuery, reachable here:
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/editdialog.html
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/edititems.html
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/editmonsters.html
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/editquests.html
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/editquestlogs.html

These editors create (and read) the resource files that are placed in res/xml .

Things that I really don't like about the editors are that they aren't really 
connected to eachother. For example, you cannot create a monster and a droplist 
for that monster at the same time, you have to do that in the two different 
editors.

Also, the conversation editor doesn't give a good overview of the dialog flow. 
I would like to see the conversation options and get a feel for the length and 
branching of each reply.

To solve this, we could implement a new editor that combines the above editors 
in some way - maybe a stand alone program (c++, qt?, java?, web based?). Or we 
could use some existing editor that other RPGs use. For map editing, we use 
Tiled, which works wonderful in my opinion. If we could use some existing 
editor for items, monsters and conversation as well, that would be really great.

Anyone know of customisable editors for monsters, items and conversation that 
we can use? Must be cross-platform. Preferably GPL.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 5 Feb 2011 at 6:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 10 Feb 2011 at 8:13
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Merged from issue 158:

A couple of candidates for editor enhancement are:



http://game-editor.com/Main_Page

and

http://rpgtoolkit.net/home/
with
http://rpgtoolkit.net/extend/plugins/read-xml/

Both open source to allow customization.

```

Original comment by `oskar.wi...@gmail.com` on 10 Feb 2011 at 8:14

**Comment by GoogleCodeExporter** [comment]
```
Issue 158 has been merged into this issue.
```

Original comment by `oskar.wi...@gmail.com` on 10 Feb 2011 at 8:15

**Comment by GoogleCodeExporter** [comment]
```
Work on the new editors have started. The new editor will be a combined 
interface where you can edit all resource types in one editor. This one will 
also be web-base, and make extensive use of jQuery.
```

Original comment by `oskar.wi...@gmail.com` on 7 Apr 2011 at 10:02
- Changed state: **Started**
- Added labels: **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
Preview version committed as part of r66 and r68.
Reachable at:
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/editor.html

```

Original comment by `oskar.wi...@gmail.com` on 11 Apr 2011 at 6:43

**Comment by GoogleCodeExporter** [comment]
```
With the conversation editor in place (committed in r72 and r73), and the 
resource loader updated to the new resource files (committed in r77), this is 
now ready for the next release.

Yay!
```

Original comment by `oskar.wi...@gmail.com` on 5 May 2011 at 6:15
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:50
- Changed state: **Done**
